### PR TITLE
endpoints section loading & application context enabling for war, jag…

### PR DESCRIPTION
**Pull request by amalkasubasinghe**
…gery, php

### CODE REVIEW

File: modules/jaggeryapps/appmgt/src/modules/application/endpoints.jag [hunk 1]
```diff
@@ -228,4 +206,34 @@ var createJsonResponse = function (restEndpoints, soapEndpoints, webEndpoints, s
...
+    webEndpoints.push({context : context, url: webUrl});
+
+    var endpoints = undefined;
+    try {
+        // check whether accessible
+        get(webUrl, {}, "text");
+
+        endpoints = createJsonResponse(null, null, webEndpoints, null);
+    } catch (e) {
+        log.info(e);
```
Comment (kasundsilva) [inline modules/jaggeryapps/appmgt/src/modules/application/endpoints.jag]: Why this is info log ?

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/home/js/home.js [hunk 2]
```diff
@@ -76,14 +76,18 @@ function generateDefaultLaunchUrl() {
...
-    // This is implemented for ESB & MSS only.
-    if (application.applicationType == "wso2esb" || application.applicationType == "mss") {
-        showLoadingEndpointView();
-        if(selectedApplicationRevision.status == APPLICATION_RUNNING){
-            var deploymentURL = generateDefaultLaunchUrl();
-            timerId = setInterval(function () {
-                loadEndpoints(deploymentURL, applicationType);
-            }, 2000);
+    // This is not implemented for wso2dataservice and mss 1.0.0 runtimes.
+    if (application.applicationType != "wso2dataservice") {
```
Comment (kasundsilva) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/home/js/home.js]: Why these values are hardcoded ?

File: modules/jaggeryapps/appmgt/src/modules/application/endpoints.jag (hunk 1 above)
Comment (amalkasubasinghe) [inline modules/jaggeryapps/appmgt/src/modules/application/endpoints.jag]: This has removed in later commit

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/home/js/home.js (hunk 2 above)
Comment (amalkasubasinghe) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/home/js/home.js]: this is a temporary condition, until we fixed loading dss endpoints

**Pull request merged**